modular_skyrat/modules/mob_container/mob_container.dm

```diff
@@ -0,0 +1,36 @@
+/mob/proc/store_mob(mob/tostore)
+	if(!istype(tostore))
+		return
+
+	if(!tostore.on_enter_container(src) || !on_container_entered(tostore))
+		return
+
+	ADD_TRAIT(tostore, TRAIT_OXYIMMUNE, REF(src)) // We give the mob inside oxy-damage immunity. Hope this doesn't come back to bite us in the ass.
```

Comment:
Can we explain why the stored mob is given immunity to such damage in the comment? Also can you change it to TRAIT_NOBREATH if it's simply an issue with the lack of breathable air.

Reply:
you should instead just redirect the breath out to the tile the mob is on

Reply:
For once, I agree. That'd be a good idea.